util/codec/codec.go

```diff
@@ -22,14 +22,15 @@ import (
 )
 
 const (
-	nilFlag byte = iota
-	bytesFlag
-	compactBytesFlag
-	intFlag
-	uintFlag
-	floatFlag
-	decimalFlag
-	durationFlag
+	nilFlag          byte = 0
```

Comment:
To make the codec flag stable, we may need to add more flag later, insert in the middle will change many existing flag values.